modules/kn-service-apply.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * serverless/knative_serving/serverless-applications.adoc
+// * serverless/cli_reference/kn-services-ref.adoc
+
+[id="kn-service-apply_{context}"]
+= Applying service declarations
+
+You can declaratively configure a Knative service by using the `kn service apply` command. If the service does not exist it will be created, otherwise the existing service will be updated with the options that have been changed.
```

Comment:
s/it will be created/is created
s/will be updated/is updated
(we avoid _will_ for some reason)